designsystem/src/main/res/color/color_selector_secondary_avon.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
the `Avon` brand have only one `color_selector_secondary` file and the others have two (one for light and other for dark) is this correct?

Reply:
Good point! I'll adjust it.